webapp/src/js/controllers/reports-content.js

```diff
@@ -21,7 +21,12 @@ var _ = require('underscore');
       'ngInject';
 
       var ctrl = this;
-      var unsubscribe = $ngRedux.connect(null, Actions)(ctrl);
+      var mapStateToTarget = function(state) {
+        return {
+          selectMode: state.selectMode
+        };
+      };
+      var unsubscribe = $ngRedux.connect(mapStateToTarget, Actions)(ctrl);
```

Comment:
I think we should already start mapping dispatch to target, even though we only have 6 actions so far.
(applies to all controllers that are currently allowed to call all actions).

Reply:
Sounds good, I think it's a bit convoluted when using `ng-redux` but it should be fine with thunks

Reply:
Convoluted? I got it working with this (for the super simplistic reducers that we currently have):
```suggestion
var mapDispatchToTarget = function(dispatch) {
  return {
    clearCancelCallback: Actions(dispatch).clearCancelCallback
  }
};
```
Re-reading this, I guess it's a bit convoluted and unnecessary at this point. 
About thunks, I imagine we are going to need them at some point anyway.

Reply:
Yep that's the approach I was thinking of too, no I think you're right it's good to add them in now it makes the logic flow a bit clearer. We probably will need thunks eventually but I guess it's best to leave adding the library until we need it so I'll just stick with this for now